Simplenote/Classes/SPNoteEditorViewController+Extensions.swift

```diff
@@ -191,6 +191,98 @@ extension SPNoteEditorViewController {
 }
 
 
+// MARK: - History
+//
+extension SPNoteEditorViewController {
+
+    /// Indicates if note history is shown on screen
+    ///
+    @objc
+    var isShowingHistory: Bool {
+        return historyViewController != nil
+    }
+
+    /// Shows note history
+    ///
+    @objc
+    func showHistory() {
+        let loader = SPHistoryLoader(note: currentNote)
```

Comment:
(Added a note below as well!) Please take a look at the [VersionsController](https://github.com/Automattic/simplenote-macos/blob/develop/Simplenote/VersionsController.swift) publisher/subscriber mechanism we've got in the Mac App.

That'd help us support multiple Editors out of the box (iPad Multiwindow), and will simplify the History Initialization